www/docs/blog/posts/2023-10-08-slsa-generation-for-your-artifacts.md

```diff
@@ -0,0 +1,229 @@
+
+GoReleaser takes the ever-growing risks in the realm of software supply chain security incredibly seriously. From the onset of this era of heightened security concerns, GoReleaser has been at the forefront, continuously adding features to safeguard your artifacts against potential software supply chain attacks such as [generating an SBOMs](https://goreleaser.com/customization/sbom/), [signing your artifacts](https://goreleaser.com/customization/docker_sign/), and more.
+
+>  _If you want to learn more about the general software supply chain security features supported by GoReleaser, check out our [blog post](https://blog.goreleaser.com/goreleaser-and-software-supply-chain-security-e8917b26924b/) on the topic._
+
+In this blog post, we will explore how GoReleaser can help you generate SLSA provenance for your artifacts and how you can leverage the slsa-github-generator to automate the process.
+
+## slsa-github-generator
+
+I would like to start with my favorite quote:
+
+> _Each of these attacks could have been prevented if there were a way to detect that the delivered artifacts diverged from the expected origin of the software. But until now, generating verifiable information that described where, when, and how software artifacts were produced (information known as provenance) was difficult. This information allows users to trace artifacts verifiably back to the source and develop risk-based policies around what they consume._ - [Improving software supply chain security with tamper-proof builds](https://security.googleblog.com/2022/04/improving-software-supply-chain.html)
+
+Currently (_unfortunately_), provenance generation is not widely supported yet but hopefully will be in the future. And this is where the slsa-github-generator comes into play. 
```

Comment:
I would phrase it as `Unfortunately, provenance generation is not widely supported yet, but hopefully will be in the future` or something like that